### ISSUE TITLE
Nested parameters

### ISSUE THREAD

**Issue by activestylus** [KaneCohen/tokenfield#13]
So there's `itemName` and `newItemName`, but apparently they both have limits.

For example, when you submit a form with `<input name="email[subject]">` the data is interpreted by the server as an object like so:

`{"email" : { "subject" : "Hello there } }`

The trouble is, when I set `itemName: "email[subject]"` I don't receive any data on the server. I believe this may be due to the fact that this plugin does not update the value of the input it is substituting. So it gets submitted as a blank.

Ideally I'd like both new and existing items to be submitted to the same parameter as well. I have no need to process or handle them differently

Is it doable?

**Comment by KaneCohen** [comment]
Thanks for the report.

First about submitting values to the same field. It is doable - you just need to set `itemName` and `newItemName` config options to the same value. It might look like this:
```js
{
  itemName: 'email[subject]',
  newItemName: 'email[subject]'
}
```
For my uses I require distinct naming for the submitted values in order to process them correctly so I went with separate submittable names.

As for the issue with the lack of data. That happens due to the fact that the field you're attaching tokenfield to is being kept within the form and it is appended at the end of the tokenfield (in order to restore the field if user destroys tokenfield insance). So since the input field has the same input as "itemName" - it overrides it with its blank state.

I'll update the library so that the original input field would be prepended instead of appended.

**Comment by KaneCohen** [comment]
Issue with empty data has been fixed in latest version (0.4.0). Input field will be prepended to the tokenfield container, so it shouldn't override tokenfield-set data.
